src/components/Nav/UserMy/UserMy.js

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
+import { IoIosArrowDown, IoIosArrowUp } from 'react-icons/io';
+
+//STYLES
+import styled from 'styled-components';
+
+function UserMy() {
+  const [isWidth, setIsWidth] = useState(false);
+  const [map, setMap] = useState([1, 2, 3, 4, 5, 6, 7, 8, 9]);
+
+  const width = () => {
+    setIsWidth(!isWidth);
+  };
```

Comment:
isWidth나 width 같은 이름으로는 해당 변수의 목적 등을 파악하기 힘들거 같습니다. 일단 이 함수는 toggle 하는거 같아서 toggleWidth 등의 이름을 생각해 볼 수 있을거 같은데 width 또한 무슨 의도로 선택하신 단어인지 알려주시면 좋을거 같아요
```suggestion
  const width = () => {
    setIsWidth(prev => !prev);
  };
```

Reply:
> isWidth나 width 같은 이름으로는 해당 변수의 목적 등을 파악하기 힘들거 같습니다. 일단 이 함수는 toggle 하는거 같아서 toggleWidth 등의 이름을 생각해 볼 수 있을거 같은데 width 또한 무슨 의도로 선택하신 단어인지 알려주시면 좋을거 같아요

아 Height 를 늘려주는 건데 Width 라고 사용했네요 지금 수정하겠습니다 
그리고 prev 는 이전꺼를 이전꺼의 반대로 해준다 이런 느낌으로 사용하는게 맞는건가요?

Reply:
@jungjun-Sung 맞습니다!